packages/browser-wallet/src/chain.ts

```diff
@@ -333,6 +318,7 @@ export class ChainWatcher implements Chain {
       } catch (error) {
         // TODO: Handle error. Likely the user rejected the login
         chainLogger.error(error);
+        console.log(error);
```

Comment:
```suggestion
```
I'm making a guess that this was a temporary convenience, and that the existing logger statement is OK on its own.

Reply:
Your guess is spot on.